Makefile

```diff
@@ -109,9 +109,11 @@ cron/data/request.pb.go: cron/data/request.proto |  $(PROTOC)
 
 generate-docs: ## Generates docs
 generate-docs: checks/checks.md
-checks/checks.md: checks/checks.yaml checks/main/*.go
+checks/checks.md: checks/docs/checks.yaml checks/docs/*.go checks/docs/generate/*.go
```

Comment:
Can we put docs in <root>/docs, instead of checks/docs. that way it is easily discoverable by anyone.

Reply:
Do you mean just the `checks.md` file or the logic to create it or both?

Reply:
I'd put everything in the same directory.

Reply:
you can keep them all, upto you or just checks.md. any way is fine. just <root>/docs is the first place to discover details. if it is hard, can skip for now.

Reply:
Done. Moved it all to `/docs`